### PR TITLE
SRCH-4874 Data Migration for Launch Support - ALL sites - Content settings

### PR DESCRIPTION
**Pull request by krbhavith**
## Summary
- Update new SERP content settings data with old content settings

### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: app/models/content_settings_updater.rb
Comment (krbhavith) [inline app/models/content_settings_updater.rb:37]: @lorawoodford is this the right way to migrate the Paperclip format file to ActiveStorage type. All the date is transfer perfectly, except byte_size (`affiliate.header_logo_blob.byte_size`) which may be reason for the broken logo image on the page.

Comment (lorawoodford) [reply app/models/content_settings_updater.rb:38]: @krbhavith Take a look at the official recommendations from the paperclip maintainers on how to migrate existing data from paperclip to ActiveStorage: https://github.com/thoughtbot/paperclip/blob/main/MIGRATING.md. We'll want to follow that lead and not roll our own approach here.

Additionally, we should probably create a separate rake task that only has responsibility for this image data migration as it has the potential to be a rather long-running task.  We will be duplicating all our existing logo images in S3 as part of this task.  We should also pull in @GSA/search-gov-devops in testing/planning for this in case they have any concerns about our ability to handle (temporarily) doubling our S3 storage usage for production logos.  The alt text data update should also be part of this separate task since it relies on the activestorage record being present first.

So:

- One task for header/footer links
- One task for image-related content

I think it would be totally appropriate to split this out into two stories as well (so as not to hold up your links work from going out), but leave that to you since you did the prior investigation and leg work.

Comment (krbhavith) [reply app/models/content_settings_updater.rb:38]: We already have ActiveStorage configuration required for this migration before. I was following this implementation https://github.com/thoughtbot/paperclip/blob/main/MIGRATING.md#moving-files-on-a-remote-host-s3-azure-storage-gcs-etc. I will try other approaches for migrating in this documentation.

Comment (krbhavith) [reply app/models/content_settings_updater.rb]: I have separated out the image content settings from this PR. Image content settings migration will be handled in. [SRCH-4961](https://cm-jira.usa.gov/browse/SRCH-4961)

**Review by lorawoodford — COMMENTED** [review]
@krbhavith This code LGTM, but I'm not yet hitting "approve" because I'm leaving a question over in JIRA about what the intended functionality is here.  Once that's answered, happy to approve.

**Review by lorawoodford — APPROVED** [review]
@krbhavith LGTM

**Pull request merged**
